### PR TITLE
logo consistency

### PR DESCRIPTION
**Pull request by phyothiha**
## Issue Reference
Default theme logo on some pages

## Description
Changing Admin and Shop logo in the admin panel does not affect on listed pages.
1. Admin Login
2. Admin Forget Password
3. Admin Reset Password
4. Customer Login
5. Customer Register
6. Customer Forget Password
7. Customer Reset Password
8. Shop Header Desktop
9. Shop Header Mobile
10. Shop Cart 
11. Shop Checkout

## How To Test This?
1. Go to Admin Panel > Settings > Channels > Select Default > Update Logo in **Design Section**
2. Go to Admin Panel > Configure > Design > Update Logo 

## Documentation
- [x] Site logos need to be consistent.

### CODE REVIEW

File: packages/Webkul/Admin/src/Resources/views/users/forget-password/create.blade.php [hunk 1]
```diff
@@ -7,10 +7,11 @@
     <div class="flex justify-center items-center h-[100vh]">
         <div class="flex flex-col gap-5 items-center">
             <!-- Logo -->
-            <img 
-                class="w-max" 
-                src="{{ bagisto_asset('images/logo.svg') }}" 
-                alt="Bagisto Logo"
+            <img
+                src="{{ core()->getCurrentChannel()->logo_url ?? bagisto_asset('images/logo.svg') }}"
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/users/forget-password/create.blade.php:11]: Admin is something that is not a channel. You need to use configuration, you can go to the 'Configure -> Design' and there you will see the admin logo, just place that key in the below method,
core()->getConfigData('...')

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Admin/src/Resources/views/users/forget-password/create.blade.php [hunk 2]
```diff
@@ -7,11 +7,19 @@
     <div class="flex justify-center items-center h-[100vh]">
         <div class="flex flex-col gap-5 items-center">
             <!-- Logo -->
-            <img 
-                class="w-max" 
-                src="{{ bagisto_asset('images/logo.svg') }}" 
-                alt="Bagisto Logo"
-            >
+            @if ($logo = core()->getConfigData('general.design.admin_logo.logo_image', core()->getCurrentChannelCode()))
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/users/forget-password/create.blade.php:10]: There should be no channel base code dependency here.

File: packages/Webkul/Shop/src/Resources/views/components/layouts/header/mobile/index.blade.php [hunk 3]
```diff
@@ -132,10 +132,9 @@ class="max-h-[30px]"
                 aria-label="@lang('shop::app.components.layouts.header.bagisto')"
             >
                 <img
-                    src="{{ bagisto_asset('images/logo.svg') }}"
-                    alt="Bagisto"
-                    width="131"
-                    height="29"
+                    src="{{ core()->getCurrentChannel()->logo_url ?? bagisto_asset('images/logo.svg') }}"
+                    alt="{{ config('app.name') }}"
+                    style="height: 30px"
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Resources/views/components/layouts/header/mobile/index.blade.php:137]: There should be no inline style. Width and height are specifically given for some reason, please revert it.

File: packages/Webkul/Admin/src/Resources/views/users/forget-password/create.blade.php [hunk 4]
```diff
@@ -7,11 +7,19 @@
...
+                <img
+                    class="w-[110px] h-10"
+                    src="{{ Storage::url($logo) }}"
+                    alt="{{ config('app.name') }}"
+                />
+            @else
+                <img
+                    class="w-max" 
+                    src="{{ bagisto_asset('images/logo.svg') }}"
+                    alt="Bagisto Logo"
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/users/forget-password/create.blade.php:20]: {{ config('app.name') }}

File: packages/Webkul/Admin/src/Resources/views/users/reset-password/create.blade.php [hunk 5]
```diff
@@ -7,11 +7,19 @@
...
+                <img
+                    class="w-[110px] h-10"
+                    src="{{ Storage::url($logo) }}"
+                    alt="{{ config('app.name') }}"
+                />
+            @else
+                <img
+                    class="w-max" 
+                    src="{{ bagisto_asset('images/logo.svg') }}"
+                    alt="Bagisto Logo"
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/users/reset-password/create.blade.php:20]: {{ config('app.name') }}

File: packages/Webkul/Admin/src/Resources/views/users/sessions/create.blade.php [hunk 6]
```diff
@@ -6,12 +6,20 @@
...
+                <img
+                    class="w-[110px] h-10"
+                    src="{{ Storage::url($logo) }}"
+                    alt="{{ config('app.name') }}"
+                />
+            @else
+                <img
+                    class="w-max" 
+                    src="{{ bagisto_asset('images/logo.svg') }}"
+                    alt="Bagisto Logo"
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/users/sessions/create.blade.php:20]: {{ config('app.name') }}

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**